### ISSUE TITLE
Google Map Does Not Display on Preconference Page

### ISSUE THREAD

**Issue by dzoladz** [alaoweb/2020.alaoweb.org#7]
**Issue:** There should be a background image of a Google Map showing the physical location of the Nationwide Hotel & Conference Center, but it's not rendered.

As this will be a virtual conference, it's a non-issue for this year. However, it would be nice to be able to use the map backdrop next year.

**Comment by dwgreen1** [comment]
This is not a background **image** but rather a rendered Google map. I am not familiar with the history of the code here, but from what I can gather, the map does not render because it was coded originally as `hackathon` rather than `preconference`, so the references aren't what they need to be.

Do I assume correctly that all references to "hackathon" are what we are calling "preconference"?

**Comment by dzoladz** [comment]
@dwgreen1 Yep. Please update those reference as you come across them. I tried to get as many of them updated as I could before committing the initial code for us to use as a starting point, but I'm sure there are references remaining in the code.
